packages/ckeditor5-engine/src/conversion/downcasthelpers.ts

```diff
@@ -991,6 +992,17 @@ export function convertRangeSelection() {
 			return;
 		}
 
+		const modelRoot = selection.getFirstPosition()!.root as RootElement;
+
+		// Don't convert selection if it is in a model root that does not have a view root (for now this is only the graveyard root).
```

Comment:
This happens when all roots are removed and there is no editable. In this case the model document selection does not have any (own) range. But years ago we decided that we always want model document selection to return a range. In this case, we return so-called default range. Normally, it is placed at the beginning of the main root. But since all roots are removed, we need to decide where to place the range. Placing it in a detached or lazy root makes, in my opinion, less sense than placing it in a graveyard. I believe that the only other sensible option would be to abandon the idea that model document selection should always have a range.

AFAIR, from what I checked, it required more changes in our database and it would be a breaking change. What's funny, `model.Document#_getDefaultRoot()` already returned a graveyard root if there are no other roots. It's just it couldn't happen in typical uses of classic editor and other single root editor types.